newIDE/app/src/EventsSheet/EventsTree/Instruction.js

```diff
@@ -146,14 +166,50 @@ const Instruction = (props: Props) => {
           }
 
           const parameterMetadata = metadata.getParameter(parameterIndex);
-          const parameterType = parameterMetadata.getType();
+          const parameterSubType = parameterMetadata.getType();
```

Comment:
We should probably at least get this logic into a `gd.ParameterMetadata.isParsableAsExpression` or something like this, or `gd.ParameteraMetadata.getParentType` or something similar.